importer/Uploader.py

```diff
@@ -243,19 +243,33 @@ def upload(self):
         self.add_claims(self.wd_item, claims, self.log)
 
     def set_wd_item(self):
-        item_exists = True if self.data["wd-item"] is not None else False
-        if item_exists:
-            item_q = self.data["wd-item"]
-            self.wd_item = self.wdstuff.QtoItemPage(item_q)
+        """
+        Determine WD item to manipulate.
+
+        In live mode, if data object has associated WD item,
+        edit it. Otherwise, create a new WD item.
+        In sandbox mode, all edits are done on the WD Sandbox item.
+        """
+        if self.live:
+            item_exists = True if self.data["wd-item"] is not None else False
+            if item_exists:
```

Comment:
Any reason to not just use `if self.data["wd-item"] is not None` directly?

Are there situations when `if self.data["wd-item"]` is false but `self.data["wd-item"] is None` is also false?